src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
+import java.util.*;
+
 public class Duke {
+    enum Command {
+        BYE,
+        LIST,
+        MARK,
+        UNMARK,
+        TODO,
+        DEADLINE,
+        EVENT,
+        DELETE
+        ;
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    Hello! I'm Duke");
+        System.out.println("    What can I do for you?");
+        System.out.println("    ____________________________________________________________");
+        boolean StillIn = true;
+        FastReader fr = new FastReader();
+        ArrayList<Task> list = new ArrayList<Task>(100);
+        while(StillIn) {
```

Comment:
I think you need to put spacing between the brackets and each `while` and each `if`. You only do not use spaces when it is a function/method declaration